content/ember-released-5-1.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Ember 5.1 Released
+authors:
+  - the-ember-learning-team
+date: 2023-06-23T00:00:00.000Z
+tags:
+  - releases
+  - '2023'
+  - version-5-x
+---
+
+Today the Ember project is releasing version 5.1 of Ember.js, Ember Data, and Ember CLI.
+
+This release kicks off the 5.2 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+The headlining feature of this release is stable [TypeScript](https://www.typescriptlang.org) support! 🎉 The framework now publishes TypeScript types built from Ember's own source code, with Ember's [strong Semantic Versioning commitment](https://emberjs.com/releases/) baked in—bringing benefits for both TypeScript and JavaScript users. For details, a couple important caveats, and a migration guide, check out [the accompanying blog post][stable-ember-types].
+
+[stable-ember-types]: https://blog.emberjs.com/stable-typescript-types-in-ember-5-1
+
+### Changes in Ember.js 5.1
+
+Ember.js 5.1 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 5.1 introduced 1 bug fix:
+
+- [#20461](https://github.com/emberjs/ember.js/pull/20461) Fix a cyclic module crash when using Ember Engines and Ember FastBoot.
+
+#### Features
+
+Ember.js 5.1 introduced 1 feature:
+
+- [#20449](https://github.com/emberjs/ember.js/pull/20449) Publish stable TypeScript types built from Ember's source.
+
+#### Deprecations
+
+Ember.js 5.1 introduced 0 deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 5.1, please review the [Ember.js 5.1.0 release page](https://github.com/emberjs/ember.js/releases/tag/v5.1.0).
+
+---
+
+## Ember Data
+
+Ember Data is the official data persistence library for Ember.js applications.
```

Comment:
potentially we could call out our addition of a Roadmap here. It isn't something we had in place for 5.1 specifically but is something we'll be utilizing going forward including with 5.1